src/Routes/ImageManager/Images.js

```diff
@@ -111,6 +130,45 @@ const Images = () => {
     });
   };
 
+  const columnNames = [
```

Comment:
This one will be re-created every time the `Image` component is re-rendered. Move it out of `Image`. Am I right?